src/core/scenario.jl

```diff
@@ -46,6 +46,7 @@ struct Scenario <: AbstractScenario
     flexible_hvac::Union{FlexibleHVAC, Nothing}
     existing_chiller::Union{ExistingChiller, Nothing}
     absorption_chiller::Union{AbsorptionChiller, Nothing}
+    backup_reliability::BackupReliability
```

Comment:
Why do we need `BackupReliability` attached to the `Scenario`? I think that we should run the reliability simulation the same way that we run the outage simulator (see https://github.com/NREL/REopt.jl/blob/cece86212568ca6171e01469eab0bac9af6eb561/test/runtests.jl#L115 for example). Rather than create a new Scenario field we could put the default arguments to the backup reliability simulation in the function signature as keyword arguments (like `microgrid_only` in `simulate_outages`). What do you think?